src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/keycloak-client/main.tf

```diff
@@ -9,7 +9,7 @@ resource "keycloak_openid_client" "main" {
   client_id     = var.client_id
   client_secret = random_password.client_secret.result
 
-  name    = "grafana"
+  name    = var.client_id
```

Comment:
Fixes https://github.com/nebari-dev/nebari/issues/2428, thank you!

Reply:
@krassowski I will open a small PR to address that later, something was going wrong with this branch when testing, so I removed that one from now